hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx

```diff
@@ -69,30 +75,29 @@ const Orders = () => {
                         </Grid>
                         <Grid item lg={12}>
                             <Grid container spacing={1} direction="row">
-                                {Array(12)
-                                    .fill("order card component")
-                                    .map((str, idx) => (
-                                        <Grid
-                                            item
-                                            lg={3}
-                                            md={4}
-                                            sm={6}
-                                            xs={12}
-                                            key={idx}
-                                        >
-                                            <div
-                                                style={{
-                                                    border: "1px solid black",
-                                                    borderRadius: "5px",
-                                                    padding: "10px",
-                                                    textAlign: "center",
-                                                    backgroundColor: "lightblue",
-                                                }}
-                                            >
-                                                {str}
-                                            </div>
-                                        </Grid>
-                                    ))}
+                                {mockPendingOrders.map((order, idx) => (
```

Comment:
Note that this card is not for all pending orders, only with orders that have status "Submitted"
If the status is "Ready for Pickup" a different card should be displayed. Could you put an if statement here to conditionally render the right card? For the Ready for Pickup one just put an empty div with some basic styling as placeholder